### DISCUSSION TITLE
TreeView Selected Item background color

### DISCUSSION THREAD

**Discussion by Vitco96** [AvaloniaUI/Avalonia#11060] [Q&A]
Hello 

I have this treeview that is inside a scrollViewer. Does anyone know how to change the background color of an selected item ? 

 <ScrollViewer Grid.Column="0"
                      Grid.Row="3"
                      Padding="5,0,5,5">

            <TreeView Items="{Binding Dop2Devices}" 
                      Background="{StaticResource TreeViewBackGround}"
                      SelectedItem="{Binding SelectedTreeViewItem}"
                      x:Name="FirstTreeView">
                
                <TreeView.ItemTemplate></TreeView.ItemTemplate>
                <TreeView.DataTemplates>
                  
                    <TreeDataTemplate DataType="models:Dop2Device" ItemsSource="{Binding Units.Values}">
                        <TextBlock FontSize="12"
                                   Foreground="{StaticResource MustardYellow}"
                                   Text="{Binding TreeViewDeviceName}"/>
                    </TreeDataTemplate>
                 
                    <TreeDataTemplate DataType="models:Dop2Unit" ItemsSource="{Binding Objects.Values}">
                        <TextBlock FontSize="12"
                                   Foreground="{StaticResource MustardYellow}"
                                   Text="{Binding TreeViewUnitName}"/>
                    </TreeDataTemplate>
                 
                    <DataTemplate DataType="models:Dop2Object">
                        <TextBlock FontSize="12" 
                                   Foreground="{StaticResource MustardYellow}"
                                   Text="{Binding TreeViewObjectName}">
                            <ToolTip.Tip>
                                <ToolTip Background="{StaticResource AppBackgroundColor}" 
                                         Foreground="{StaticResource MustardYellow}">
                                    <TextBlock Text="{Binding ItemDescription}" 
                                               FontSize="14"/>
                                </ToolTip>
                            </ToolTip.Tip> 
                        </TextBlock>
                    </DataTemplate>
                  
                </TreeView.DataTemplates>
            </TreeView>
        </ScrollViewer>

Thank you

**Comment by timunie** [comment]
Check out original Style. Mind the version of Avalonia you use (for 0.10.x you need to change branch)

**Comment by DominicPicard** [comment]
I also had to change the color for the selected item of a TreeView. In my case I wanted to hide the selection.

You need to create a style just for your treeview and replace the the color for the "triggers " that you want

See last message in https://github.com/AvaloniaUI/Avalonia/discussions/12183
